fl4health/clients/nnunet_client.py

```diff
@@ -157,7 +157,20 @@ def __init__(
         # Some nnunet client specific attributes
         self.dataset_id: int = dataset_id
         self.dataset_name = convert_id_to_dataset_name(self.dataset_id)
-        self.dataset_json = load_json(join(nnUNet_raw, self.dataset_name, "dataset.json"))
+        try:
+            self.dataset_json = load_json(join(nnUNet_raw, self.dataset_name, "dataset.json"))
+        except Exception as e:
+            self.dataset_json = load_json(join(nnUNet_preprocessed, self.dataset_name, "dataset.json"))
+            e1 = e
```

Comment:
Maybe a dumb question, but why not just do
`except Exception as e1:`
in the catch above, rather than doing this set?

Reply:
hmm...perhaps the of e is limited? It's just a touch awkward feeling, but perhaps this is the "canonical" way to do this.

Reply:
the linters were complaining as I believe the as exists only within the scope of the except. However in testing I realized we can't actually have two except blocks with the same exception

Reply:
I like the refactor. Thanks for looking into it.